python/neutron-understack/neutron_understack/neutron_understack_mech.py

```diff
@@ -81,7 +84,60 @@ def update_network_precommit(self, context):
         pass
 
     def update_network_postcommit(self, context):
-        pass
```

Comment:
:-1: Don't do this here. Register a function to be called on `resources.SEGMENTS` and `events.AFTER_CREATE` and `events.BEFORE_DELETE` 

Here's an example in the OVN mechanism... https://github.com/openstack/neutron/blob/60c572acb18e3181d93cf6cc60920362f264aa6b/neutron/plugins/ml2/drivers/ovn/mech_driver/mech_driver.py#L624
https://github.com/openstack/neutron/blob/60c572acb18e3181d93cf6cc60920362f264aa6b/neutron/plugins/ml2/drivers/ovn/mech_driver/mech_driver.py#L301